src/tools/__tests__/snapshotSerializers/astSerializer.ts

```diff
@@ -1,24 +1,26 @@
 import { ASTNode, print } from "graphql";
-import { Plugin, Config, Refs, Printer } from "pretty-format";
+import { Plugin } from "pretty-format";
 
-export = {
-  test(value: any) {
+const plugin: Plugin = {
+  test(value) {
     return value && typeof value.kind === "string";
   },
 
   serialize(
     value: ASTNode,
-    config: Config,
-    indentation: string,
-    depth: number,
-    refs: Refs,
-    printer: Printer
+    _config,
```

Comment:
this is prob a dumb question, but if the params are unused, why not delete them instead of per-fixing them with `_`

Reply:
They don't need to be here (besides `_config`). I had originally left them before we actually had type safety as a hint, but now that we're using the `Plugin` type over asserting it, I dont feel strongly about keeping them 🤷‍♂️ Will remove